src/styles/footer/footerBottom.scss

```
@@ -0,0 +1,174 @@
+.footer-bottom {
+  padding: 50px 120px 35px;
+  position: relative;
+
+  @media (min-width:320px) {
+    display: flex;
+    flex-direction: column;
+    padding: 0;
+  }
+
+  @media (min-width:768px) {
+    max-width: 660px;
+  }
+
+  @media (min-width:1280px) {
+    max-width: 1170px;
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-gap: 100px;
+    padding: 0 0 35px;
+
+    &::after {
+      position: absolute;
+      top: 0;
+      left: 50%;
+      content: "";
+      height: 260px;
+      width: 1px;
+      background-color: rgba(26, 90, 76, 0.5);
+    }
+  }
+}
+
+.date {
+  padding: 100px 0 40px;
+  color: #0f0e08;
+  font-weight: 300;
+  font-family: $plexSansFontFamily;
+  display: flex;
+  flex-direction: column-reverse;
+  justify-content: space-between;
+  text-align: center;
+
+  @media (min-width:768px) {
+    flex-direction: row;
+  }
+
+  &__copyright {
+    padding-top: 14px;
+  }
+}
+
+.left-side {
+  display: flex;
+  justify-content: space-between;
+
+  @media (min-width:320px) {
+    margin: 110px 0;
+    flex-direction: row;
+  }
+
+  @media (min-width:768px) {
+    padding: 0 100px;
+  }
+
+  @media (min-width:1280px) {
+    margin: 0;
+    padding: 0;
+  }
+
+  
+}
+
+.icons {
+  display: flex;
+  flex-direction: column;
+
+  @media (min-width:320px) {
```

Винеси breakpoint-и в окремі змінні
```suggestion
  @media (min-width:320px) {
```